### ISSUE TITLE
gtk4/libadwaita support?

### ISSUE THREAD

**Issue by binarynoise** [linuxmint/mint-l-theme#11]
Now that the Mint-X and Mint-Y themes are updated to support gtk4/libadwaita, when will the Mint-L themes follow?

**Comment by JosephMcc** [comment]
Mint-X and Y support Gtk4. They do not support libadwaita. Those are two separate things.

**Comment by binarynoise** [comment]
@JosephMcc 
Ok, I'll edit my question

**Comment by binarynoise** [comment]
Mint-X and Y support Gtk4 for some time now.
When do you plan to add support to Mint-L for Gtk4?

**Comment by mz1193** [comment]
I agree, it would be nice to see these themes updated with GTK4 support.

**Comment by Skywalkerjet3D** [comment]
i agree as well, apps like gnome clock just look horrendously out of place

**Comment by daemonspudguy** [comment]
I also concur with this, especially since Mint-Y doesn't have a dark panel on Xfce while Mint-L does.

**Comment by JosephMcc** [comment]
> When do you plan to add support to Mint-L for Gtk4?

There is no plan. The L stands for legacy. These themes will likely go away at some point.
